File: leetcode/leetcode_cpp/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int res = 0;
        for (int i=s.size()-1; i>=0; --i) {
            if (res && isspace(s[i]))
                break;
            if (!isspace(s[i]))
                res += 1; 
        }
        return res;        
    }
};

File: leetcode/leetcode_py/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s = s.strip()
        res = 0
        for i in range(len(s) - 1, -1, -1):
            c = s[i]
            if c == ' ':
                break
            else:
                res += 1

        return res

        